### PR TITLE
[minor] checked punctuation, grammar, descriptive links

### PR DESCRIPTION
**Pull request by pnlph**
Minor correctness check and more salient, decriptive links.

Here are some standard [guidelines on writing hyperlinks](https://www.nngroup.com/articles/writing-links/), maybe an excerpt about how to write usable links could be included in the style guide?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,51 +3,50 @@
...
 
 #### Old versions of Agda
 
 If you're using an old version of Agda, you can download the corresponding version
 of the standard library on the [Agda wiki](http://wiki.portal.chalmers.se/agda/pmwiki.php?n=Libraries.StandardLibrary).
-The index for older versions of the library can be found at, for example, https://agda.github.io/agda-stdlib/v0.17/
-where 0.17 can be replaced as required.
+The module index for older versions of the library is also available. For example,
+version 0.17 can be found at https://agda.github.io/agda-stdlib/v0.17/, just
+replace in the URL 0.17 for the version that you need.
```
Comment (MatthewDaggitt) [inline README.md]: I think this would read better as `just replace 0.17 in the URL with the version you need`?

File: README.md [hunk 2]
```diff
@@ -3,51 +3,50 @@
...
-- You can browse the library's source code in glorious clickable html
-[here](https://agda.github.io/agda-stdlib/README.html).
+- You can [browse the library's source code](https://agda.github.io/agda-stdlib/README.html)
+in glorious clickable HTML.
 
-- Finally you can get an overview of the entire library by looking at the
-[index](https://agda.github.io/agda-stdlib/) which lists every module it contains
-(minus any deprecated modules).
+- Finally, you can get an overview of the entire library by looking at the
+[module index](https://agda.github.io/agda-stdlib/), which lists all modules
```
Comment (MatthewDaggitt) [inline README.md]: I think the original "[index]" reads better than the new "[module index]"

**Comment by MatthewDaggitt** [comment]
> maybe an excerpt about how to write usable links could be included in the style guide?

Hmm the style guide is about how to write Agda code rather than the documentation for the repository. Normally library contributors write very little markdown.

File: README.md (hunk 1 above)
Comment (pnlph) [reply README.md]: OK, will change.

File: README.md (hunk 2 above)
Comment (pnlph) [reply README.md]: Yes, it may be more succint in this case, the word module appears also later in the phrase. Will change.

**Pull request merged**